astropy/nddata/tests/test_utils.py

```diff
@@ -5,7 +5,8 @@
 from numpy.testing import assert_allclose
 
 from ...tests.helper import pytest
-from ..utils import extract_array, add_array, subpixel_indices
+from ..utils import (extract_array, add_array, subpixel_indices,
+                     block_reduce, block_replicate)
 
 test_positions = [(10.52, 3.12), (5.62, 12.97), (31.33, 31.77),
```

Comment:
Could you please add a scikit-image import here in a try/except, and if the import fails, skip the tests that require scipy?

An example of how to do that is at: https://github.com/astropy/astropy/blob/321a41ca6671e896dfa9e957c4a0fea80d751ad7/astropy/coordinates/tests/test_matching.py#L20